src/App.tsx

```diff
@@ -1,61 +1,152 @@
+import { useState } from 'react';
 import './App.scss';
+import 'bulma/css/bulma.css';
 
```

Comment:
Consider importing only necessary styles from Bulma instead of the whole library to reduce bundle size. 📦